components/typography/Heading.tsx

```diff
@@ -0,0 +1,89 @@
+enum HeadingTypeStyle {
+  xl = 'heading-xl',
+  lg = 'heading-lg',
+  md = 'heading-md',
+  md_semibold = 'heading-md-semibold',
```

Comment:
you should use camelCase naming convention here, because now these are constants and will be used same as variables.